src/components/dropdown-color/index.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import { translate } from 'react-i18next';
+import Dropdown from '../dropdown/';
+import style from '../dropdown/style.styl';
+
+const opts = {
+    //Dropdon Type
+    type: "color",
+    //Display item text
+    showText: false,
+    //Display selected item label
+    showLabel: true,
+    //Dropdown actions
+    actions: ["More colors..."]
+}
+
+export default (props) => (<Dropdown config={opts} {...props}/>)
```

Comment:
this is small enough so that you export it from dropdown, it makes it easier to ship later.

instead of `export default` you do:
`export class DropdownColor extends React.Component {`